src/utils.js

```diff
@@ -0,0 +1,60 @@
+//#region > Functions
+/**
+ * @returns {values} Params selected from a query
+ * @description This function gets params out of a query
+ */
+function toParams(query) {
+  const q = query.replace(/^\?/, "");
+
+  return q.split("&").reduce((values, param) => {
+    const [key, value] = param.split("=");
+
+    values[key] = value;
+
+    return values;
+  }, {});
+}
+
+/**
+ * @returns {query} A query for requests
+ * @description This function builds a query out of params
+ */
+function toQuery(params, delimiter = "&") {
+  const keys = Object.keys(params);
+
+  return keys.reduce((str, key, index) => {
+    let query = `${str}${key}=${params[key]}`;
+
+    if (index < keys.length - 1) {
+      query += delimiter;
+    }
+
+    return query;
+  }, "");
+}
+
+/**
+ * @returns {string} A guid string
+ * @description This function generates a random guid string
+ */
+function GuidGenerator() {
```

Comment:
This function should also be lowercase as all others.
Do not forget to change this in the whole source code.